src/main/java/cloud/eppo/BaseEppoClient.java

```diff
@@ -56,7 +56,8 @@ protected BaseEppoClient(
       @NotNull String apiKey,
       @NotNull String sdkName,
       @NotNull String sdkVersion,
-      @Nullable String host,
+      @Nullable String apiBaseUrl,
+      @Deprecated @Nullable String apiHost, // To be removed in v4. use `apiBaseUrl` instead.
```

Comment:
took me a minute to figure out that you renamed `host` to `apiHost` and deprecated it. I'd recommend undoing the rename (keep `host` to avoid confusion) and add the comment as standard javadocs above the method